Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/FeedProcessing/ProcessorOptions.cs

```diff
@@ -22,5 +21,7 @@ internal class ProcessorOptions
         public DateTime? StartTime { get; set; }
 
         public TimeSpan RequestTimeout { get; set; } = CosmosHttpClient.GatewayRequestTimeout;
+
+        public FeedRangeInternal FeedRange { get; set; }
```

Comment:
No need for it to be FeedRangeInternal?

```suggestion
        public FeedRange FeedRange { get; set; }
```